src/server/check-answers/index.test.js

```diff
@@ -27,10 +27,8 @@ const {
   origin,
   destination,
   licence,
-  identification,
-  biosecurity,
   'biosecurity-map': biosecurityMap
-} = validApplicationState
+} = validApplicationState.application
```

Comment:
We don't need this additional nesting - the validApplicationState **is** what's stored under `req.yar.get('application')`